connector/src/main/java/com/google/cloud/hive/bigquery/connector/config/RunConf.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.hive.bigquery.connector.config;
+
+import org.apache.hadoop.conf.Configuration;
+
+/**
+ * Class that gathers all the configuration keys and default values for the Hive-BigQuery connector.
+ */
+public class RunConf {
```

Comment:
Can we rename this class to RunConfiguration instead of RunConf?

Reply:
How about `HadoopConfig`, since most of those values are actually fetched from Hadoop configuration?

Reply:
Also, I'm proposing `HadoopConfig` instead of `HadoopConfiguration` to be consistent with `BigQueryConfig` and `BigQueryProxyConfig`.

Reply:
`HadoopConfig` works for me